include/asyncio/stream.h

```diff
@@ -9,12 +9,50 @@
 #include <asyncio/selector/event.h>
 #include <asyncio/noncopyable.h>
 #include <asyncio/task.h>
+#include <sys/socket.h>
+#include <unistd.h>
 #include <netdb.h>
 #include <utility>
 #include <vector>
-#include <unistd.h>
+
+#if defined(__APPLE__) || defined(__CYGWIN__) || defined(__NetBSD__)
+    #include <sys/ioctl.h>
+    #include <fcntl.h>
```

Comment:
`F_GETFL` defined in fcntl.h，but  ‘F_SETFL’ was not declared in this scope，move `#include <fcntl.h>` to the global include section?